src/components/ProseMirrorInner.tsx

```diff
@@ -38,15 +39,34 @@ export function ProseMirrorInner({
   mount,
   ...editorProps
 }: ProseMirrorProps) {
+  const { reactEventPlugin, registerEventListener, unregisterEventListener } =
+    useReactEventPlugin();
```

Comment:
If this is feeling verbose, you could grab the `reactEventPlugin` and capture the rest (e.g. `...eventContextValues`) to spread in below.